Tests/SwiftDocCTests/Model/RenderContentMetadataTests.swift

```diff
@@ -157,6 +157,47 @@ class RenderContentMetadataTests: XCTestCase {
 
         try assertRoundTripCoding(renderedTable)
     }
+
```

Comment:
Should we also test the extra behavior in the Codable implementation where the `alignments` value is not emitted at all if they're all `unset`?

Reply:
This is effectively tested in the existing `testRenderingTableSpans` test, at least the sense that the decoded representation is ensured to be the same. I could extend it to double-check that the decoded `alignments` property is `nil`, if that would be useful.